src/commands/build.rs

```diff
@@ -28,6 +24,9 @@ pub struct BuildCommand {
     /// Run the application after building.
     #[clap(name = BUILD_UP_OPT, short = 'u', long = "up")]
     pub up: bool,
+
+    #[clap(requires = BUILD_UP_OPT)]
+    pub up_args: Vec<OsString>,
```

Comment:
Is this intended to be visible to users running `spin build --help`, or only used programmatically?  I'm not quite sure how it fits in, sorry.

Reply:
This change was introduced by #352 — basically, `spin build --up` needs the same options as `spin up`.

Reply:
Yes, its just a different implementation of the existing behavior that made the split in `spin up` easier.